README.md

```diff
@@ -80,6 +80,14 @@ As of 2019, Pillow development is
     </tr>
 </table>
 
+## Overview
+
+The Python Imaging Library adds image processing capabilities to your Python interpreter.
+
+This library provides extensive file format support, an efficient internal representation, and fairly powerful image processing capabilities.
+
+The core image library is designed for fast access to data stored in a few basic pixel formats. It should provide a solid foundation for a general image processing tool.
+
```

Comment:
I'd move this up, right below the title. Here users need to scroll a lot to find it.